languagetool-language-modules/eo/src/main/resources/org/languagetool/rules/eo/grammar.xml

```diff
@@ -1961,59 +1962,91 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
-        </token>
-        <marker>
-          <token postag="[ROAT] nak.*" postag_regexp="yes">
-            <exception postag="T .*e" postag_regexp="yes"/>
-            <exception regexp="yes">(?-i)[A-ZĈĜĤĴŜ].*|sola</exception>
-          </token>
-        </marker>
-      </pattern>
-      <message>Ĉu vi forgesis akuzativon post transitiva verbo «\2»?</message>
-      <example correction="">Mi forgesis la <marker>akuzativo</marker>.</example>
-      <example>Mi forgesis la <marker>akuzativon</marker>.</example>
-      <example>Mi kantas kun amiko.</example>
-      <example>Mi klarigis kial li ne venis.</example>
-      <example>Mi trovis tie novan veturilon.</example>
-      <example>Ŝi faris lin idealisto.</example>
-      <example>Li sentis sin fremdulo.</example>
-      <example>Mi sentas ke li timas.</example>
-      <example>Tiu romano povas esti uzata kiel lernolibro.</example>
-      <example>Mi venos, diris li.</example>
-      <example>Mi ne scias sed mi kredas.</example>
-      <example>La lernejo en kiu li instruis estis fermita pro la krizo.</example>
-      <example>Ni entreprenas – li konfesis.</example>
-      <example correction="">Mi volas aĉeti la <marker>komputilo</marker>.</example>
-      <example>Mi volas esti la instruisto.</example>
-      <example>Vi povas kontroli ĉu Esperanto funkcias.</example>
-      <example>Mi dankas Barbara.</example>
-      <example>Kiun rolon laŭ vi havis la teamo ?</example>
-      <example>Tion oni nomas religio.</example>
-      <example>Tie loĝas mia amiko.</example>
-      <example>Kie loĝas la homo?</example>
-      <example>En la akvo ludas knaboj</example>
-      <example>Post vetero malbela lumas suno denove.</example>
-      <example>Ju pli ni havas des pli ni deziras.</example>
-      <example>Nenio pli teruras ol tertremo.</example>
-      <example>Jen la ĉambro en kiu manĝas centoj da infanoj.</example>
-      <example>Kion vi konsideras vera?</example>
-      <example>Li loĝas tie sola.</example>
-      <example>Mi trovas tion inteligenta.</example>
-      <example>Ni havis maltro por manĝi.</example>
-      <example>Li studis kiamaniere la birdoj flugas.</example>
-    </rule>
+    <rulegroup id="AKUZATIVO" name="ĝenerala problemo pri akuzativo">
+        <rule id="AKUZATIVA_TIKORELATIVO_RILATA_AL_NEAKUZATIVA_SUBSTANTIVO" name="Akuzativa “Ti”-korelativo rilata al neakuzativa substantivo.">
+            <pattern>
+                <token postag='V.* tr.*' postag_regexp='yes'></token>
+                <token postag='T.* akz.*' postag_regexp='yes'></token>
+                <marker>
+                    <token postag='O.* nak.*' postag_regexp='yes'></token>
+                </marker>
+            </pattern>
+            <message>Aspektas ke ‘<match no="2"/>’ rilatas al ‘<match no="3"/>’, sed ‘<match no="3"/>’ ne akuzativas: probable devintus esti ‘<suggestion><match no="3"/>n</suggestion>’.</message>
```

Comment:
I have to admit that I did not really have put large care in the error message itself. So sure, « Ŝajnas, ke » perfektly suits me.
I mainly speak Esperanto at home with my wife, so my Esperanto tends to drift from the official one ☺
I will make a patch for this.